internal/server/utils/redisCmdAdapter_test.go

```diff
@@ -284,6 +386,18 @@ func TestParseWebsocketMessage(t *testing.T) {
 			expectedCmd:  "JSON.INGEST",
 			expectedArgs: []string{"", "$..field", `{"field":"value"}`},
 		},
+		{
+			name:         "Test simple Q.WATCH command",
+			message:      "q.watch \"select $key, $value where $key like 'k?'\"",
+			expectedCmd:  "Q.WATCH",
```

Comment:
Shouldn't this be QWATCH? Why Q.WATCH?

Reply:
We have renamed the command to be consistent with GET.WATCH, ZRANGE.WATCH, etc.

Reply:
QWATCH has been renamed to Q.WATCH everywhere in the codebase in this PT #1102

Reply:
Done.